src/chain.h

```diff
@@ -182,7 +182,7 @@ class CBlockIndex
     //!
     //! Note: this value is modified to show BLOCK_OPT_WITNESS during UTXO snapshot
     //! load to avoid the block index being spuriously rewound.
-    //! @sa RewindBlockIndex
+    //! @sa NeedsRedownload
```

Comment:
How is this change different from  fa49430914fd16fbc34078bd8929b6ef635a8c42 in #21789 ?